website/src/hooks/useFontDownload.tsx

```diff
@@ -1,22 +1,26 @@
 import { useEffect, useState } from "react";
 
 import { MetadataProps } from "../@types/[font]";
+import { fontsourceDownload } from "../utils/fontsourceUtils";
 
 const useFontDownload = (metadata: MetadataProps, downloadLink: string) => {
   const [fontLoaded, setFontLoaded] = useState(false);
 
   useEffect(() => {
     setFontLoaded(false);
 
-    // Fetch font file
-    new FontFace(metadata.fontName, `url(${downloadLink})`)
-      .load()
-      .then((result) => {
-        document.fonts.add(result);
-        setFontLoaded(true);
-      })
-      .catch((error) => console.log(error));
-  }, [metadata.fontId, metadata.fontName, downloadLink]);
+    fetch(downloadLink)
+      .then((res) => res.text())
+      .then((cssFile) => {
+        document.getElementById("font-preview-css").innerText =
+          cssFile.replaceAll(
```

Comment:
I have a feeling the replaceAll function might be the culprit for the iOS Safari incompatibility. It's really new.

Reply:
Yeah, I just realized that when I tried viewing it on a mac to check for the same problem.

Reply:
Figured out the problem was caused by something else, but I still changed this for compatibility.